lua/telescope/builtin/internal.lua

```diff
@@ -330,8 +330,8 @@ end
 
 internal.vim_options = function(opts)
   -- Load vim options.
-  local vim_opts = loadfile(utils.data_directory() .. path.separator .. 'options' ..
-                            path.separator .. 'options.lua')().options
+  local vim_opts = loadfile(utils.data_directory() .. Path.path.sep .. 'options' ..
+                            Path.path.sep .. 'options.lua')().options
```

Comment:
I think i already commented this :rofl: but: can we do `Path:new({utils.data_directory(), 'options', 'options.lua'}):absolute()` or so